Rebalances pickaxes (and most pickaxe derivatives) force, sharpness, armor penetration, and wounding mods. Makes skeletons slightly more vulnerable to mining tools in the process.

TemporalOroboros opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
- Makes all* pickaxes (including drills) sharp. Pickaxes are pointy, drills are sharp.
- Gives all pickaxes some degree of armor piercing.
- Gives pickaxes a bonus to inflicting wounds. Specifically just barely enough for basic pickaxes to inflict compound fractures on a `BIO_JUST_BONE` bodypart with a preexisting hairline fracture.
- Gives drills the same force per tier as pickaxes. Also makes them significantly weaker against armor but significantly stronger against bare flesh.
- Makes the survival pen capable of inflicting basic wounds to better reflect its status as a mining tool.
- Makes skeletons slightly more vulnerable to mining tools.
- Gives the sonic jackhammer a significant nerf to force and a significant buff to its wound inflicting abilities.
- Rearranges the pickaxe descriptions because they were defined at the end of the vars for some reason.
- Changes the pickaxe attack verbs a bit for flavor.

*This does not include the sonic jackhammer and its derivative(s) because they use soundwaves and if the oontz could puncture flesh my head would be a sponge.

<details>
<summary>Balance Changes:</summary>

- Pickaxes:
  - armor piercing: 0 --> 25
  - sharpness: `SHARP_NONE` --> `SHARP_POINTY`
  - wound_bonus: 0 --> 25
- Silver Pickaxe:
  - force: 17 --> 18
- Diamond Pickaxe:
  - force: 19 --> 20 (can now break lockers)
- Improvised Pickaxe:
  - armor piercing: 0 --> 15
  - sharpness: `SHARP_EDGED` --> `SHARP_POINTY`
  - wound_bonus: 0 --> 5
  - bare_wound_bonus: 0 --> 10
- Drills:
  - force: 15 --> 18
  - armor piercing: 0 --> -40
  - sharpness: `SHARP_NONE` --> `SHARP_EDGED`
  - wound_bonus: 0 --> -20
  - bare_wound_bonus: 0 --> 50
- Diamond Drills:
  - force: 15 --> 20 (can now break lockers)
  - armor piercing: 0 --> -25
- Sonic Jackhammer:
  - force: 15 --> 5
  - armor piercing: 0 -> 80
  - wound_bonus: 0 --> 50
  - bare_wound_bonus: 0 --> 50
- Demonic Jackhammer:
  - force: 15 --> 20 // (can now break lockers) \:)
- Survival Pen:
  - force: 3 -> 5
  - armor piercing: 0 --> 10
  - wound_bonus: 0 --> 10
  - bare_wound_bonus: 0 -- > 12 // Just barely enough to inflict basic piercing wounds.

</details>

<details>
<summary>How wounds are applied:</summary>

## How wounds are applied

When a carbon mob takes damage (either `BRUTE` or `BURN`) it [foists most of the damage handling off onto its external bodyparts](https://github.com/tgstation/tgstation/blob/b000d5bbd966545714cd42d9333a5c1f842697ae/code/modules/mob/living/carbon/damage_procs.dm#L3-L46). When the bodypart [receives the damage](https://github.com/tgstation/tgstation/blob/b000d5bbd966545714cd42d9333a5c1f842697ae/code/modules/surgery/bodyparts/_bodyparts.dm#L209-L280) the first thing it does it apply whatever damage modifiers are applicable to the incoming damage. Then it starts handling wounds.

### Wounding type:

The types of wounds applied to the bodypart depend on four factors:

- The primary damage type of the incoming damage. Essentially the greater between the  incoming `BRUTE` and `BURN` damage.
  - Primarily `BRUTE` damage deals either `WOUND_BLUNT`, `WOUND_SLASH`, or `WOUND_PIERCE` type wounds depending on the other three factors.
  - Primarily `BURN` damage always deals `WOUND_BURN` type wounds.
- The sharpness of the incoming damage, usually as dictated by the attacking objects sharpness var.
  - No sharpness makes `BRUTE` dominated damage attempt to inflict `WOUND_BLUNT` type wounds.
  - Edged sharpness makes `BRUTE` dominated damage attempt to inflict `WOUND_SLASH` type wounds.
  - Pointy sharpness makes `BRUTE` dominated damage attempt to inflict `WOUND_PIERCE` type wounds.
- The biostate of the bodypart. Currently either `BIO_JUST_BONE` or `BIO_FLESH_BONE`.
  - `BIO_JUST_BONE` bodyparts convert whatever incoming `BRUTE` sourced wounding type to `WOUND_BLUNT` as there is no flesh to slash or pierce. However, if the incoming wounding type was not already `WOUND_BLUNT` a multiplier is applied to the incoming wounding power.
    - If the wounding type was initially `WOUND_SLASH` the incoming wounding damage is multiplied by `0.6`.
    - If the wounding type was initially `WOUND_PIERCE` the incoming wounding damage is multiplied by `0.75`.
  - `BIO_FLESH_BONE` bodyparts usually don't try to convert wounding types, whether they do is dependent on how mangled the bodypart is.
    - How mangled a bodypart is is dependent on the wounds it has:
      - Critical wounds of any type except dismemberment or `WOUND_BONE` mangle the flesh. This is only relevant if the bodypart is `BIO_FLESH_BONE`.
      -  Critical `WOUND_BONE` type wounds mangle the bones. This is only relevant when rolling for dismemberment.
    - Specifically, if the flesh is mangled the bodypart will convert the wounding type to `WOUND_BLUNT` with the same wounding damage multipliers as a `BIO_JUST_BONE` bodypart.

### Wounding power:

After the wounding type and the base wounding power (equal to damage and modified if converting sharp wounds to blunt wounds) are determined the bodypart then starts to [roll for wounds](https://github.com/tgstation/tgstation/blob/b000d5bbd966545714cd42d9333a5c1f842697ae/code/modules/surgery/bodyparts/_bodyparts.dm#L364-L444).

The severity and probability of the wounds applied to the bodypart are dependent on a number of factors. The first of these is the base wounding power of the check.

The base wounding damage of the check starts as equal to the incoming brute or burn damage of the attack causing the check. This is first modified by the bio state of the bodypart as detailed above. After this it is further modified dependent on traits possessed by the mob or bodypart. As these traits are not applicable to your average crewmember I will be disregarding them and focusing on the default modifiers to wounding damage. The first modifier reduces the base wounding damage to no more than `35`. This is to prevent extremely high damage weapons, such as the d-sword, from inflicting objectively ridiculous base wounding damages. The wounding damage is then raised to an [exponent](https://github.com/tgstation/tgstation/blob/2161005a4489ebd5912373cf855b1f53351640eb/code/__DEFINES/wounds.dm#L4) defined, as of the current version, to be `1.4`. A random integer is then rolled between 1 and the resulting base wounding damage. This constitutes the base wounding power.

After the base wounding power is calculated the [sum of a number of modifiers](https://github.com/tgstation/tgstation/blob/b000d5bbd966545714cd42d9333a5c1f842697ae/code/modules/surgery/bodyparts/_bodyparts.dm#L446-L488) is added to it:
- The first modifier is the total wound armor of all of the clothing on the bodypart. This is subtracted from the total wounding power.
- The second modifier is the bare wound bonus relevant to the wounding check. This is equal to the `bare_wound_bonus` of the attacking weapon in the most relevant case.
  - If there is any applicable wound armor at all this is not applied.
  - Otherwise this value is added to the total wounding power.
- The third modifier is the wounding power relevant to the wounding check. This is equal to the `wound_bonus` of the attacking weapon in the most relevant case.
- The fourth modifier is the sum of the threshold penalties of all of the existent wounds. This is added to the total wounding power.
- The fifth modifier is the wound resistance of the bodypart. This is subtracted from the total wounding power.
- The sixth modifier is the disabled wound penalty of the bodypart. This is skipped if the total damage on the bodypart is any less than the maximum damage of the bodypart. Otherwise it is added to the total wounding power. We will be disregarding this value as if your bodypart is already at max damage wounds are pretty much redundant.

In all the total wounding power is equal to:
- A random number between 1 and ((the relevant damage of the attack * biostate wound conversion multipliers)  ** 1.4)
- - The sum of all relevant wound armor
- + The bare wound bonus of the attacking object if there is not armor.
- + The wound bonus of the attacking object regardless of armor.
- + The threshold penalties of the existing wounds.
- - The wound resistance of the bodypart. (Usually 0 so I will be disregarding it)

### Applying wounds:

Once the wounding type and the wounding power is tabulated the bodypart then attempts to apply the relevant wound. In all relevant cases this is the most severe wound of the wounding type that has a wounding threshold below the wounding power and is more severe than the any existing wound of that type. If no such wound exists no wound is applied.

</details>

<details>
<summary>The wounding potency of the mining tools:</summary>

All of the following are calculated assuming no armor, no wound-modifying traits, no wound resistance, and no disabled wound bonus.

<details>
<summary>Mining Tool Stats:</summary>

Mining Tool: | Force: | Armor Pen: | Sharpness: | Wound Bonus: | Bare Wound Bonus:
-------------|-------|--------------|-----------|-----------------|------------------
Pickaxe | 15 | 30 | `SHARP_POINTY` | 25 | 0
Silver Pickaxe | 18 | 30 | `SHARP_POINTY` | 25 | 0
Diamond Pickaxe | 20 | 30 | `SHARP_POINTY` | 25 | 0
Rusty Pickaxe | 1 | 30 | `SHARP_POINTY` | 25 | 0
Compact Pickaxe | 10 | 30 | `SHARP_POINTY` | 25 | 0
Improvised Pickaxe | 10 | 15 | `SHARP_POINTY` | 5 | 10
Emergency Disembarkation Tool: | 15 | 30 | `SHARP_POINTY` | 60 | 0
Power Drill | 18 | -40 | `SHARP_EDGED` | -20  | 50
Diamond Power Drill | 20 | -25 | `SHARP_EDGED` | -20 | 50
Sonic Jackhammer | 5 | 80 | `SHARP_NONE` | 50 | 50
Demonic Jackhammer | 20 | 80 | `SHARP_NONE` | 50 | 50
Survival Pen: | 5 | 10 | `SHARP_POINTY` | 10 | 12

</details>

<details>
<summary>Wound Stats: </summary>

Wound: | Type: | Severity: | Wound Threshold: | Wound Penalty: | Mangles Flesh:
--------|------|---------|---------------------|-----------------|------------
Minor Breakage | `WOUND_PIERCE` | `MODERATE` | 30 | +20 | No
Open Puncture | `WOUND_PIERCE` | `SEVERE` | 50 | +35 | No
Ruptured Cavity | `WOUND_PIERCE` | `CRITICAL` | 100 | +50 | Yes
Rough Abrasion | `WOUND_SLASH` | `MODERATE` | 20 | +10 | No
Open Laceration | `WOUND_SLASH` | `SEVERE` | 50 | +25 | No
Weeping Avulsion | `WOUND_SLASH` | `CRITICAL` | 80 | +40 | Yes
Joint Dislocation | `WOUND_BLUNT` | `MODERATE` | 35 | +15 | No
Hairline Fracture | `WOUND_BLUNT` | `SEVERE` | 60 | +30 | No
Compound Fracture | `WOUND_BLUNT` | `CRITICAL` | 115 | +50 | No

</details>
</details>

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
- Pickaxes are essentially sledgehammers with railroad spikes welded to the end. It makes no sense that they are unable to pierce flesh.
- They are meant to punch through solid granite and magnetite, they should punch through armor too.
  - Apparently drills get stuck on fabric easily, who knew? They now have trouble penetrating armor.
  - The sonic jackhammer uses _sound_ and should not be mitigated that well by armor designed to stop knives, bullets, and lasers.
- The desc variables were just weirdly ordered, now they're not.
- The attack verbs are slightly more flavorful.

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
balance: Skeletons have been found to be slightly vulnerable to excavation equipment.
Nanotransen's product design department has corrected some issues with their standard mining tools:
balance: The pickaxes were insufficiently pointy, they have been sharpened.
balance: A mix-up in supply lines meant the drills were being constructed with smooth bits meant for sanding. This has been corrected and drills are now being constructed with the correct, serrated drill bits.
balance: The frequency of the sonic jackhammer has been tuned to pass through soft tissues more efficiently and with less significant harm.
balance: The survival pens were being manufactured with their caps on backwards. A technician uncommented the relevant code.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

TemporalOroboros committed:
pointy

- Gives all pickaxes (other than jackhammers) `SHARP_POINTY`.
- Rearranges a couple vars.
- Gives all pickaxes wound_bonus 10.
- Changes pickaxe attack verbs.

GunHog commented:
Does this mean that I can now mine someone's **Brainite**?

Ryll-Ryll commented:
Tbqh you could probably put the wound bonus up to like 20, piercing wounds aren't nearly as threatening point-for-point as slashing wounds

TemporalOroboros committed:
Merge branch 'master' into pointy-pickaxes

TemporalOroboros committed:
Buffs wound bonuses and damages

TemporalOroboros commented:
@Ryll-Ryll 20 wasn't enough to make pickaxes break bones so I set it to 30. I also buffed the damage for pickaxes slightly.

TemporalOroboros commented:
Alright, the buff change has not shown up yet. Weird.

TemporalOroboros committed:
Reorders two variables

TemporalOroboros commented:
Hopefully this fixes whatever git's doing with my commits.

TemporalOroboros commented:
Git finally updated the PR.

IndieanaJones commented:
Mining cyborg shadow buffs???

TemporalOroboros commented:
Sure, they are drills after all. I'm not sure why they didn't have the same damage buffs as the pickaxes in the first place.

Winterous commented:
I think drills would be more of a slashing wound than a piercing, because they don't do their thing instantly, but rather held down as it's thrust at the target; there'll be a little movement and it'll 'dance' around their flesh. The difference with the surgical drill is that mining drills are big and conical, instead of a pencil.

Also I suppose you could add an attack_self to pickaxes to toggle between pointed and blunt, the silver and diamond ones DO have a blunt side after all, you could flip the sprite around.

TemporalOroboros commented:
@Winterous I think think that you're right. Drills should probably be sharp since they would inflict more widespread damage than the pickaxe. I'll do that.
As for giving a pickaxes the ability to swap between blunt and pointy, they can actually deal blunt wounds once they have dealt enough piercing wounds which honestly makes more sense (and is less snowflake).

Winterous commented:
Is there a way to make them not become less effective at breaking bones, like most weapons? There's something like that on the victim end, Trait_easy_wound or something.

TemporalOroboros committed:
Adds Winterous' suggestion.

Drills are now `SHARP_EDGED` instead of `SHARP_POINTY`

TemporalOroboros commented:
There only thing that determines how effective weapons are at dealing wounds (on the weapons end) are the weapons force, damage type, sharpness, wound_bonus, and bare_wound_bonus.

- force: Increases the likelihood that a weapon will inflict wounds. This affects all wounds equally.
- damage type: Determines whether the weapon inflices `BRUTE` (bone, slash, pierce) or `BURN` wounds.
- sharpness: Determines what kind of `BRUTE` wounds the weapon inflicts.
- wound bonus: Increases the likelihood of inflicting wounds on all target. This affects all wounds equally.
- bare wound bonus: Increases the likelihood of inflicting wounds on targets without wound armor. This affects all wounds equally.

To note is that if the bodypart being hit either doesn't have flesh or has mangled flesh the effective damage is reduced and any `BRUTE` damage object switches to dealing bone wounds regardless of sharpness.

TemporalOroboros commented:
There is no currently existent way to make something sharp not worse at dealing blunt wounds (that I know of) and I really don't want to feature creep what was supposed to be a simple balance PR any further.

Ryll-Ryll commented:
All looks fine to me

necromanceranne commented:
If you wanted to make the pickaxe slightly more useful as a weapon while keeping the upper damage a little lower, you could give it some armor piercing. That's entirely realistic to how a pick would hurt you.

TemporalOroboros commented:
I think I might do that actually.

TemporalOroboros committed:
Adds armour penetration

- Pickaxe armour penetration: 0 --> 15
- Drill armour penetration: 0 --> 15
- Jackhammer armour penetration: 0 --> 25

Winterous commented:
With armour penetration they might be a little overtuned, particularly the drills.
Diamond drill is suddenly a larger, more damaging circular saw with armour penetration; shit's gonna be the go-to for weaponry hands down.
I think the drills (other than jackhammer) should have negative AP if anything. There's a reason arborists wear chainmail, the teeth get snagged easily; by similar principle a drill is going to have a tough time getting through hardy fabrics like kevlar or exosuits, and it's designed for rock so steel plate will be rough as well.

So drills should have -10/-15 armour pen, making a helmet and vest reduce their damage from 15/20 to 8-9/11-12, but they will absolutely massacre anyone unprotected; this would also encourage going for limb attacks instead of chest or head.

*
Jackhammer can keep its armour pen, but I think the damage should probably be toned down; remember that blunt attacks to chest have a chance to knockdown, and to the head can rattle your brain making you confused. Combined with its respectable wound bonus, shit's probably gonna cause concussions pretty well. In terms of mechanics, this thing isn't so much a jackhammer as it is a sonic weapon; it's still gonna have impact, but most of the work is the sound waves.

Damage, 12-15 would probably be more balanced; shattering bones will make up for it I'm sure.
The chance of knocking down / knocking someone senseless with a blunt attack is force, a straight 20% chance to knock someone down is pretty crazy, and current some of the only things that approach that are syndicate toolbox and diamond pickaxe; but neither have these wound bonuses or armour penetration.

TemporalOroboros committed:
Winterous's suggestions:

- Drills have been given negative armor penetration both to represent them getting stuck on fabric and to prevent severe overtuning.
- The sonic jackhammer has had its force halved. I've also tuned the armor pen and wound bonus values way way _way_ up tocement it as a wound specialist.
- The demonic jackhammer gets to keep the boosted force because there's only one of the things.

TemporalOroboros commented:
@Winterous Done. Drills have negative armor pen (-15 for normal -10 for diamond). I've also halved the force on the sonic jackhammer. In exchange I boosted the armor pen and wound bonus values for it significantly. It might be a wound specialist now. I've left the boosted force on the demonic frost miners version mostly because it only shows up on icebox and when it does there's only ever one of them.

TemporalOroboros committed:
Boosts pickaxe armor pen a bit.

- Increases basic pickaxe armor pen from 15 to 25.

Winterous commented:
Holy christ that's just gonna explode people's bones, nice.

ATH1909 commented on code/modules/mining/equipment/mining_tools.dm:
```diff
@@ -85,24 +94,36 @@
+	armour_penetration = 50
+	sharpness = SHARP_NONE
+	wound_bonus = 40
+	bare_wound_bonus = 60 // It is pumping rock-shattering levels of ultrasound directly into your body. That's going to do _nasty_ things to your bones.
```
um

ATH1909 replied:
two hits against an armorless region with this thing will causes a guaranteed compound fracture

and said compound fracture is pretty likely to happen with just ONE hit, especially with the demonic frost miner version

TemporalOroboros replied:
Yes, yes it will.

TemporalOroboros replied:
It's a tool meant to blast apart cubic meters of rock using ultrasound being applied directly to your body, that's not going to be good for your skeleton. I could nerf the wound bonuses and damage slightly, but the lower I make the damage the higher the wound bonuses would need to be for it to have a chance of dealing a compound fracture.

Tlaltecuhtli commented:
miner's CBT arsenal update

ATH1909 commented on code/modules/mining/equipment/mining_tools.dm:
```diff
@@ -16,8 +15,12 @@
+	attack_verb_continuous = list("hits", "pierces", "mines", "attacks")
+	force = 15
+	armour_penetration = 25
+	wound_bonus = 30
```
this is high enough to guarantee at least a basic puncture wound on the first hit, and at least a second degree puntue wound on the second hit.

TemporalOroboros replied:
I wanted the pickaxes to be able to work their way through the entire stack of puncture wounds and bone wounds, since they are pointy tools meant for breaking rock. This means that they need a significant bonus to wounding power. When I did the math this came out to at least 21 wound bonus for a compound fracture assuming that there was already a critical puncture wound (required for a `SHARP_PIERCING` object to deal bone wounds) and a _prefect wounding roll_. I upped it to 30 when I came to the conclusion that a 1/(20^1.4) chance was too low.

ATH1909 commented on code/modules/mining/equipment/mining_tools.dm:
```diff
@@ -47,32 +50,38 @@
 	toolspeed = 0.3
-	desc = "A pickaxe with a diamond pick head. Extremely robust at cracking rock walls and digging up dirt."
-	force = 19
+	force = 20
```
can these break open lockers now?

TemporalOroboros replied:
Yes, so can thrown spears and those are even easier to get.

Winterous replied:
Yeah keep in mind that diamond pickaxes are only available from the mining rewards vendor, and while a busy miners might be able to buy 15 of them in a round, they do take a reasonable amount of work to get a hold of.

ATH1909 commented on code/modules/mining/equipment/mining_tools.dm:
```diff
@@ -85,24 +94,36 @@
+	bare_wound_bonus = 60 // It is pumping rock-shattering levels of ultrasound directly into your body. That's going to do _nasty_ things to your bones.
 
 /obj/item/pickaxe/improvised
 	name = "improvised pickaxe"
```
I was debating whether or not to mention this, but improvised pickaxes will inherit the wound-related stuff from normal pickaxes.

is that okay?

TemporalOroboros replied:
I'm reasonable okay with that. even if they are improvised they are still pickaxes after all.

TemporalOroboros replied:
Alright, I made the improvised pickaxe a bit closer to a kitchen knife.

Wallemations commented:
Does any of this apply to the survival pen-pick?

TemporalOroboros commented:
@Wallemations The what now?

TemporalOroboros commented:
I just realized that bone-only people exist and that pickaxes and drills might not be able to even theoretically deal compound fractures against them at this point because they have the 0.75/0.6 base wounding damage multiplier for `SHARP_POINTY`/`SHARP_EDGED` objects without the possibility of getting a critical piercing/slashing wound to boost the wounding damage. I don't want to push the wounding damage _even higher_ because it's honestly high enough as it is. Does anyone have any advice?

TemporalOroboros commented:
Hold on, I just had an idea. I'll tweak the stats a bit.

TemporalOroboros committed:
I _really_ hate `BIO_JUST_BONE`

- Makes pickaxes anti-armor specialists
- Makes drills relatively ineffective against armor, but more effective against bare skin.
- Reduces the sonic jackhammers damage. Increases its armor pen and wound bonus. Nerfs its bare wound bonus.

TemporalOroboros commented:
Bodyparts with `BIO_JUST_BONE` redirect all incoming brute wounds to `WOUND_BLUNT` similar to a mangled `BIO_FLESH_BONE` bodypart would. This, ironically, makes them harder to deal bone wounds to because while they reduce any incoming sharp wounding damage by the same amount a mangled `BIO_FLESH_BONE` bodypart would they cannot have a critical sharp or piercing wound contributing +40 or +50 wounding power to the pool respectively.

Since I want pickaxes and drills to break bones effectively I would like them to be able to inflict critical bone wounds. A critical bone wound (compound fracture) requires a total wounding power of 115 to be applied. A severe bone wound (hairline fracture) contributes 30 wounding power to the pool, reducing the total wounding power necessary from the pickaxe/drill to 85. Assuming that the bodypart has `BIO_FLESH_BONE` the bodypart must have a critical piercing/slashing wound to inflict a bone wound. These wounds contribute +50/+40 wounding power reducing the total necessary wounding power from the drill to +35/+45. Assuming the pickaxe/drill is a basic pickaxe/power drill it will have a force of 15/18. Reduced by the relevant factor when converting from piercing/slashing damage to blunt damage (0.75/0.6) this is effectively 11.25/10.8. To the power of 1.4 and floored to calculate wounding power and the effective wounding power of the pickaxe/drill from force alone is 29/27. Assuming a max roll to wound the required total wound bonus of the pickaxe/drill is (35-29)/(45-27) or 6/18. This is a reasonable value, and I'd probably give them wound bonuses in this ballpark.... _however_.

Bio state `BIO_JUST_BONE` exists! This has all the same force reducing effects when translating sharp damage to blunt damage and also cannot receive piercing/slashing wounds! This removes the wound power supplied by the critical piercing/slashing wounds in the last calculation and blasts the required total wounding bonus of the pickaxe/drill from 6/18 to _56_/_58_. These are patently absurd numbers.

In conclusion, I hate `BIO_JUST_BONE`.

TemporalOroboros committed:
Buffs the survival pen, nerfs the improvised pickaxe.

- The survival pen can now inflict basic piercing and bone wounds.
- The improvised pickaxe now more accurately represents its construction.

TemporalOroboros commented:
@Wallemations The survival pen can now cause the basic bleeding wounds.

TemporalOroboros committed:
Converts overcompenstation for `BIO_JUST_BONE` into passing tool behaviour down the attack chain.

- Makes [/obj/item/bodypart/proc/receive_damage] take tool behaviour into account.
- Nerfs overly large wound bonuses.

Winterous commented:
Those certainly are all words.

I'm pretty sure that a survival knife, with a force of 15, wound of -5, and bare wound of 10, is capable of decapitating the skeletons dropped by legions.
If that can do it, surely these things are capable of it too.

Are you perhaps overlooking that skeleton type humans tend to have easy_dismember, or whatever it's called, removing the sharp damage penalty for bone wounds?

Kaffe-work commented:
once again, has anyone seen a pickaxe irl? they don't swing fast, nor are they very weildy, but I would not enjoy getting hit by one. the fact that it's comparable to a knife is pretty laughable.

Winterous commented:
Yeah, the fact that we don't really model unwieldiness outside of "this needs two hands to use" is iffy.
I suppose you could make them attack more slowly? The cleaving saw has a fast attack mode.

`/obj/item/melee/transforming/cleaving_saw/melee_attack_chain(mob/user, atom/target, params)
	..()
	if(!active)
		user.changeNext_move(CLICK_CD_MELEE * 0.5) //when closed, it attacks very rapidly`

So maybe add that to pickaxes, make them 1.5x cooldown?

optimumtact commented:
This PR trims a tree in the middle of a forest, mining tools need a complete overhaul and rethink so that there are appropriately differentiated categories, the mining pickaxe is pretty much entirely vestigal at this point, material versions included.

Winterous commented:
Rip :(

ATH1909 commented:
rip
